features/step_definitions/secrets.js

```diff
@@ -79,6 +79,8 @@ module.exports = function server() {
         }).then((response) => {
             this.jobId = response.body[0].id;
             this.secondJobId = response.body[1].id;
+            if (2 in response.body) this.thirdJobId = response.body[2].id;
```

Comment:
Yeah, this is the LIST jobs endpoint so `response.body` is an array of jobs. 

I agree with Petey on not using `in` here.